src/BusinessTable.jsx

```diff
@@ -0,0 +1,151 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { Table, Thead, Tbody, Tr, Th, Td } from "@chakra-ui/react";
+
+const BusinessTable = () => {
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+
+  const getData = async () => {
+    try {
+      const response = await backend.get('/business');
+      setData(response.data);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  });
```

Comment:
this is good that you are rendering the data upon the web page loading with a useEffect. however, try adding a console.log statement here and open up your network/console tab in inspect element. you will see that useEffect will constantly always be running every single render of this page, which is not what we want. 

we want to beable to run the useEffect one time only, so the way that we set this up is with an empty dependency array. if you dont understand what i mean, your line 21 should look like so:

}, []);

this means that it will only render once on startup. when you add stuff into your dependency array, that means that the useEffect will only trigger upon the change of the variable within the dependency array.